docs/docs/guide/account.md

```diff
@@ -0,0 +1,21 @@
+---
+id: account
+title: Account & Network Settings
+slug: /account
+---
+
+## Adding a new account
+
+In order to use another account that is derived from your seed phrase to Freighter, you can do the following:
+
+- From the account balances page, you can click on the "accounts" dropdown from the identicon in the upper left corner of the screen in order to open the account options.
+- From the options page, you can "Import a Stellar secret key" if you already have your secret key or you can "Create new Stellar wallet" if you want to generate one. Please note - when you are importing a new secret key, you are essentially resettings your wallet and your new key will not be derived from your existing seed phrase.
```

Comment:
Hm, I'm not sure I understand what's meant by "resetting your wallet".

Reply:
fixed in b7bf492967be38cc5bb34aa663335e6bec51ed0f